code/_globalvars/lists/flavor_misc.dm

```diff
@@ -122,6 +122,71 @@ GLOBAL_LIST_INIT(ghost_forms_with_accessories_list, list(
 ))
 //stores the ghost forms that support hair and other such things
 
+// A mapping between AI_EMOTION_* string constants, which also double as user readable descriptions, and the name of the iconfile. (used for /obj/machinery/status_display/ai )
+GLOBAL_LIST_INIT(ai_status_display_emotes, list(
```

Comment:
I think these two should be in their own file, probably under the ai folder. These misc files aren't good

Reply:
Good idea.  Applied with https://github.com/tgstation/tgstation/pull/75740/commits/631ed4606d3fa3fd9809ec6c60e96d40e7678102